File: wwwroot/js/dto/task/GetAllTasksResponse.js
export class GetAllTasksResponse {
  constructor({ id, title, description, dueDate, statusId, createdBy, createdByUser, createdAt, updatedAt, assignedUsers }) {
    this.id = id;
    this.title = title;
    this.description = description;
    this.dueDate = dueDate;
    this.statusId = statusId;
    this.createdBy = createdBy;
    this.createdByUser = createdByUser;
    this.createdAt = createdAt;
    this.updatedAt = updatedAt;
    this.assignedUsers = assignedUsers;
  }
}

File: Source/Infrastructure/Adapters/Input/DTOs/Task/GetAllTasksResponse.cs
using System;
using System.Collections.Generic;
using TaskBoardDemo.Source.Infrastructure.Adapters.Input.DTOs.User;

namespace TaskBoardDemo.Source.Infrastructure.Adapters.Input.DTOs.Task;

public class GetAllTasksResponse
{
    public Guid Id { get; set; }
    public string Title { get; set; } = string.Empty;
    public string? Description { get; set; }
    public DateTime? DueDate { get; set; }
    public Guid StatusId { get; set; }
    public Guid CreatedBy { get; set; }
    public GetUserByIdResponse CreatedByUser { get; set; } = null!;
    public DateTime UpdatedAt { get; set; }
    public List<GetUserByIdResponse> AssignedUsers { get; set; } = new List<GetUserByIdResponse>();
}